src/shared/utils/animationFrameUtils.ts

```diff
@@ -41,6 +41,7 @@ export function requestAnimationTimeout(
   return frame
 }
 
+// small wrapper function around cancelAnimationFrame to use with requestAnimationTimeout
```

Comment:
If the function body is small and not complicated, then I think it doesnt warrant an explanation as it will typically be of low value.